kubernetes/loculus/values.yaml

```diff
@@ -1457,21 +1457,60 @@ defaultOrganisms:
         - name: "Nextclade (EV-D68)"
           url: "https://clades.nextstrain.org/?input-fasta={{[unalignedNucleotideSequences:EV-D68+rich|fasta]}}&dataset-name=community/hodcroftlab/enterovirus/enterovirus/linked/EV-D68&dataset-server=https://raw.githubusercontent.com/alejandra-gonzalezsanchez/nextclade_data/multi-pathogen-evs/data_output"
       metadataAdd:
-        - name: clade
-          displayName: Clade
+        - name: clade_cv_a16
+          displayName: Clade CV-A16
+          header: "Clade"
           noInput: true
+          generateIndex: true
+          autocomplete: true
+          initiallyVisible: true
+          includeInDownloadsByDefault: true
+          preprocessing:
+            args: 
+              segment: CV-A16
+            inputs: {input: nextclade.clade}
+        - name: clade_cv_a10
```

Comment:
@alejandra-gonzalezsanchez sorry I forgot to mention this - you need to create a separate clade for each enterovirus as well as preprocessing needs to know which segment's nextclade processing results to use